packages/file_selector/file_selector_windows/lib/src/dart_file_open_dialog_api.dart

```diff
@@ -0,0 +1,83 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ffi';
+
+import 'package:ffi/ffi.dart';
+import 'package:win32/win32.dart';
+
+/// FileOpenDialogAPI provider, it used to interact with an IFileOpenDialogInstance.
+class FileOpenDialogAPI {
+  /// Sets dialog options.
+  int setOptions(int options, IFileOpenDialog dialog) {
+    return dialog.setOptions(options);
+  }
+
+  /// Returns dialog options.
+  int getOptions(Pointer<Uint32> ptrOptions, IFileOpenDialog dialog) {
+    return dialog.getOptions(ptrOptions);
+  }
+
+  /// Set confirmation button text on dialog.
```

Comment:
Here and a few other places: use a third-person verb -- in this case "Sets".

https://dart.dev/guides/language/effective-dart/documentation#prefer-starting-function-or-method-comments-with-third-person-verbs